test/decorators/regular_event_decorator_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RegularEventDecoratorTest < ActiveSupport::TestCase
+  include ActionView::TestCase::Behavior
+
+  def setup
+    ActiveDecorator::ViewContext.push(controller.view_context)
+    @regular_event = ActiveDecorator::Decorator.instance.decorate(regular_events(:regular_event1))
+    @finished_regular_event = ActiveDecorator::Decorator.instance.decorate(regular_events(:regular_event8))
+  end
+
+  test 'holding_cycles' do
```

Comment:
```suggestion
  test '#holding_cycles' do
```

modelやdecoratorなどメソッド名そのままのテストの場合はクラスメソッドかインスタンスメソッドがわかるようにこういう感じにしていただけるとありがたいです〜

（user_decoratorもこうなってはいないのですが、別Issueで対応したいと思います。）